Simplex/Solver.php

```diff
@@ -41,6 +43,14 @@ function getSteps()
 
 
 
+	/** @return int */
+	function getMaxSteps()
```

Comment:
This getter is IMO not necessary since the variable is instantiated and (eventually) filled in constructor.